src/Microsoft.AspNet.Mvc.TagHelpers/Internal/GlobbingUrlBuilder.cs

```diff
@@ -127,7 +127,8 @@ private IEnumerable<string> FindFiles(IEnumerable<string> includePatterns, IEnum
 
             var matches = matcher.Execute(_baseGlobbingDirectory);
 
-            return matches.Files.Select(ResolveMatchedPath);
+            return matches.Files.Select(ResolveMatchedPath)
+                .OrderBy(path => path, new PathComparer());
```

Comment:
instead of always doing `new PathComparer()` just create a static default variable:
```C#
private static readonly PathComparer DefaultPathComparer = new PathComparer();
...
.OrderBy(path => path, DefaultPathComparer);
```